app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>All Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+  <h2><%= contestant.name %></h2>
```

Comment:
Could use labes here to show Contestant Name and Project.
Otherwise, really great job with rendering all this data!